src/formatDate.js

```diff
@@ -8,7 +8,36 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let resultDate = '';
+  const objectDateFormat = {};
+  const dateSplitted = date.split(fromFormat[fromFormat.length - 1]);
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    const less = dateSplitted[i] < 30;
+    const more = dateSplitted[i] >= 30;
```

Comment:
The variables 'less' and 'more' are not necessary and can lead to confusion. The comparison should be done directly in the if conditions when checking the year format conversion. Additionally, the variable names 'less' and 'more' do not clearly indicate their purpose. Consider using more descriptive names or performing the checks directly in the if statements.